cli/cook/subcommands/ssh.py

```diff
@@ -49,6 +49,6 @@ def ssh(clusters, args, _):
 
 def register(add_parser, _):
     """Adds this sub-command's parser and returns the action function"""
-    parser = add_parser('ssh', help='ssh to Mesos agent by job or instance uuid')
+    parser = add_parser('ssh', help='ssh to Mesos agent or container by job or instance uuid')
```

Comment:
Why are we leaving the word Mesos here but removing it from `ls`?

Reply:
Technically a Mesos agent is not the same as a container. But I'm ok just having container only

Reply:
Yeah, that sounds good